wasm_vm_test.cc

```diff
@@ -1,9 +1,23 @@
 #include "include/proxy-wasm/wasm_vm.h"
 
 #include "gtest/gtest.h"
+#include "include/proxy-wasm/null.h"
+#include "include/proxy-wasm/null_vm_plugin.h"
 
 namespace proxy_wasm {
-namespace {
+
+class TestNullVmPlugin : public NullVmPlugin {
+public:
+  TestNullVmPlugin() = default;
+  ~TestNullVmPlugin() override = default;
+};
+
+TestNullVmPlugin *test_null_vm_plugin_ = nullptr;
+RegisterNullVmPluginFactory register_test_null_vm_plugin("test_null_vm_plugin", []() {
+  auto plugin = std::make_unique<TestNullVmPlugin>();
+  test_null_vm_plugin_ = plugin.get();
```

Comment:
We need to capture the variable in the lambda.

Reply:
We don't because it is a global.  Do you think we should for style reasons?

Reply:
I think it's OK. Nit: renaming the variable to make it look like a global variable

Reply:
Done.